src/pages/index.js

```diff
@@ -284,6 +284,9 @@ const IndexPage = () => {
         <p>
           <a href="/contact#imprint">Impressum</a> gem&auml;&szlig; &sect; 5 TMG.
         </p>
+        <p>
+          <a href="/contact">Contact</a>
```

Comment:
This link is literally three lines above.

Reply:
Ah yeah but Impressum !== Contact. I never thought it would be there